modules/cassandra/src/main/java/org/testcontainers/cassandra/CassandraContainer.java

```diff
@@ -0,0 +1,168 @@
+package org.testcontainers.cassandra;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.testcontainers.cassandra.delegate.CassandraDatabaseDelegate;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.ext.ScriptUtils;
+import org.testcontainers.ext.ScriptUtils.ScriptLoadException;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.File;
+import java.net.InetSocketAddress;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Optional;
+
+/**
+ * Testcontainers implementation for Apache Cassandra.
+ * <p>
+ * Supported image: {@code cassandra}
+ * <p>
+ * Exposed ports: 9042
+ */
+public class CassandraContainer extends GenericContainer<CassandraContainer> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("cassandra");
+
+    private static final Integer CQL_PORT = 9042;
+
+    private static final String DEFAULT_LOCAL_DATACENTER = "datacenter1";
+
+    private static final String CONTAINER_CONFIG_LOCATION = "/etc/cassandra";
+
+    private static final String USERNAME = "cassandra";
+
+    private static final String PASSWORD = "cassandra";
+
+    private String configLocation;
+
+    private String initScriptPath;
+
+    public CassandraContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        addExposedPort(CQL_PORT);
+
+        withEnv("CASSANDRA_SNITCH", "GossipingPropertyFileSnitch");
+        withEnv("JVM_OPTS", "-Dcassandra.skip_wait_for_gossip_to_settle=0 -Dcassandra.initial_token=0");
+        withEnv("HEAP_NEWSIZE", "128M");
+        withEnv("MAX_HEAP_SIZE", "1024M");
+        withEnv("CASSANDRA_ENDPOINT_SNITCH", "GossipingPropertyFileSnitch");
+        withEnv("CASSANDRA_DC", DEFAULT_LOCAL_DATACENTER);
+    }
```

Comment:
One more thing related to the wait strategies, the following tests `testCassandraQueryWaitStrategy`, `testInitScriptWithRequiredAuthentication` declare `CassandraQueryWaitStrategy`, so, Can I assume if we make default using `waitFor(new CassandraQueryWaitStrategy())` it will work when authentication is enabled and disabled, right?

Reply:
@eddumelendez Sure, I can put this strategy by default and all the tests should pass, even without specified strategy, whether the authentication is enabled or not.

Reply:
let's doing like that 👍🏽

Reply:
@eddumelendez Changes pushed. I let you review them.